### ISSUE TITLE
Textbox font-family inconsistency between browsers

### ISSUE THREAD

**Issue by edvardsniedre** [eBay/skin#1450]
Tested on Skin v12.5.0 both DS4 and DS6

I have been working on a11y task where someone reported that in Safari input placeholder text has not sufficient color contrast.

Then I realised that Chrome and Safari displays fonts for `textbox` inconsistently. 

Chrome on left and Safari on right.
![Screenshot 2021-05-19 at 21 31 25](https://user-images.githubusercontent.com/7050068/118880369-b0eaca00-b8e9-11eb-98aa-5210b4d9e6ca.png)

I have investigated this and it looks like that this could be due to that `input.textbox__control` is missing `font-family: inherit;` thats why it picks up `system-ui` font instead in some browsers.

![Screenshot 2021-05-19 at 21 34 21](https://user-images.githubusercontent.com/7050068/118880651-01622780-b8ea-11eb-8903-f281426c0290.png)

I have looked in `textbox.less` and could see that `textarea.textbox__control` does have `font-family: inherit;` and is consistent between browsers.

Chrome on left and Safari on right.
![Screenshot 2021-05-19 at 21 03 03](https://user-images.githubusercontent.com/7050068/118878270-4769bc00-b8e7-11eb-8759-92cf01f8389c.png)
![Screenshot 2021-05-19 at 21 22 21](https://user-images.githubusercontent.com/7050068/118879226-5ac95700-b8e8-11eb-963e-af66dfe45d51.png)

**Comment by ianmcburnie** [comment]
Thank you @edvardsniedre for the great investigation and write up! We will fix.

**Comment by ianmcburnie** [comment]
Fixed in 12.6.0
